packages/firestore/lite/src/api/reference.ts

```diff
@@ -108,10 +108,10 @@ export class DocumentReference<T = DocumentData> extends _DocumentKeyReference<
 
   constructor(
     readonly firestore: FirebaseFirestore,
-    readonly converter: FirestoreDataConverter<T> | null,
+    _converter: FirestoreDataConverter<T> | null,
```

Comment:
Why does this need to have its `readonly` property removed? I don't see the converter value being assigned to a new value anywhere.

Reply:
The readonly is already part of the parent class, which already uses the underscore naming.